src/app/features/home/details/details.page.ts

```diff
@@ -379,16 +386,21 @@ export class DetailsPage {
   }
 
   private openCaptureClub() {
-    return this.activeDetailedCapture$
+    return combineLatest([
+      this.activeDetailedCapture$,
+      this.diaBackendAuthService.token$,
+    ])
```

Comment:
```suggestion
    return combineLatest([
      this.activeDetailedCapture$.pipe(
        concatMap(
          activeDetailedCapture => activeDetailedCapture.diaBackendAsset$
        ),
        isNonNullable()
      ),
      this.diaBackendAuthService.token$,
    ])
```

It appears that the first `concatMap` after `first()` operator could be removed. The whole pipe would be a bit simpler this way.